Dangerfile

```diff
@@ -0,0 +1,82 @@
+
+has_app_changes = !git.added_files.grep(/Source.*\.swift/).empty?
+has_test_changes = !git.modified_files.grep(/Tests/).empty?
+
+###
+### Specific files to watch
+###
+
+# CI related files to watch for changes.
+@DANGER_CI_CD_FILES = ['.travis.yml', 'Dangerfile']
+
+# Dependency files to watch for changes.
+@DANGER_DEPENDENCY_FILES = ['Gemfile', 'JSQDataSourcesKit.podspec']
+
+# determine if any of the files were modified
+def didModify(files_array)
+	did_modify_files = false
+	files_array.each do |file_name|
+		if git.modified_files.include?(file_name) || git.deleted_files.include?(file_name)
+			did_modify_files = true
+		end
+	end
+	return did_modify_files
+end
+
+### WARNINGS
+
+warn('Changes to CI/CD files') if didModify(@DANGER_CI_CD_FILES)
+warn('Changes to dependency related files') if didModify(@DANGER_DEPENDENCY_FILES)
+warn("Big PR, try to keep changes smaller if you can") if git.lines_of_code > 500
```

Comment:
let's say: "This is a large pull request! Can you break it up into multiple smaller ones instead?"

Reply:
Done.